test/guardian/db_test.exs

```diff
@@ -125,6 +125,6 @@ defmodule Guardian.DBTest do
     )
 
     assert Guardian.DB.revoke_all(sub) == {:ok, 3}
-    assert Repo.all(Token.query_schema()) == []
```

Comment:
This function was public solely for the sake of testing which is a pretty big anti-pattern. I updated these tests accordingly.